components/library/objfw/Makefile

```diff
@@ -0,0 +1,58 @@
+#
+# CDDL HEADER START
+#
+# The contents of this file are subject to the terms of the
+# Common Development and Distribution License (the "License").
+# You may not use this file except in compliance with the License.
+#
+# You can obtain a copy of the license at usr/src/OPENSOLARIS.LICENSE
+# or http://www.opensolaris.org/os/licensing.
+# See the License for the specific language governing permissions
+# and limitations under the License.
+#
+# When distributing Covered Code, include this CDDL HEADER in each
+# file and include the License file at usr/src/OPENSOLARIS.LICENSE.
+# If applicable, add the following below this CDDL HEADER, with the
+# fields enclosed by brackets "[]" replaced with your own identifying
+# information: Portions Copyright [yyyy] [name of copyright owner]
+#
+# CDDL HEADER END
+#
+
+#
+# Copyright (c) 2023, Jonathan Schleifer. All rights reserved.
+#
+
+BUILD_BITS = 32_and_64
```

Comment:
Why do we need 32-bit?

Reply:
To allow linking 32 bit and 64 bit applications to it. Should 32 bit no longer be built at all?

Reply:
32 bit support is being slowly phased out.  So if there is no immediate need for it, then it should be avoided.

Reply:
Removed.